src/results/ashp.jl

```diff
@@ -0,0 +1,126 @@
+- `thermal_to_load_series_mmbtu_per_hour`  # Thermal power production to serve the heating load series [MMBtu/hr]
+- `thermal_to_storage_series_ton` # Thermal production to ColdThermalStorage
+- `thermal_to_load_series_ton` # Thermal production to cooling load
+- `electric_consumption_series_kw`
+- `annual_electric_consumption_kwh`
+- `annual_thermal_production_tonhour`
+
+
+!!! note "'Series' and 'Annual' energy outputs are average annual"
+	REopt performs load balances using average annual production values for technologies that include degradation. 
+	Therefore, all timeseries (`_series`) and `annual_` results should be interpretted as energy outputs averaged over the analysis period. 
+
+"""
+
+function add_ashp_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
+    r = Dict{String, Any}()
+    r["size_ton"] = round(value(m[Symbol("dvSize"*_n)]["ASHP"]) / KWH_THERMAL_PER_TONHOUR, digits=3)
+    @expression(m, ASHPElectricConsumptionSeries[ts in p.time_steps],
+        p.hours_per_time_step * sum(m[:dvHeatingProduction][t,q,ts] / p.heating_cop[t][ts]
+        for q in p.heating_loads, t in p.techs.ashp) 
+    ) 
+
+    @expression(m, ASHPThermalProductionSeries[ts in p.time_steps],
+        sum(m[:dvHeatingProduction][t,q,ts] for q in p.heating_loads, t in p.techs.ashp)) # TODO add cooling
+	r["thermal_production_series_mmbtu_per_hour"] = 
+        round.(value.(ASHPThermalProductionSeries) / KWH_PER_MMBTU, digits=5)
+	r["annual_thermal_production_mmbtu"] = round(sum(r["thermal_production_series_mmbtu_per_hour"]), digits=3)
+
+	if !isempty(p.s.storage.types.hot)
+        @expression(m, ASHPToHotTESKW[ts in p.time_steps],
+		    sum(m[:dvHeatToStorage][b,"ASHP",q,ts] for b in p.s.storage.types.hot, q in p.heating_loads)
+        )
+        @expression(m, ASHPToHotTESByQualityKW[q in p.heating_loads, ts in p.time_steps], 
+            sum(m[:dvHeatToStorage][b,"ASHP",q,ts] for b in p.s.storage.types.hot)
+        )
+    else
+        @expression(m, ASHPToHotTESKW[ts in p.time_steps], 0.0)
+        @expression(m, ASHPToHotTESByQualityKW[q in p.heating_loads, ts in p.time_steps], 0.0)
+    end
+	r["thermal_to_storage_series_mmbtu_per_hour"] = round.(value.(ASHPToHotTESKW) / KWH_PER_MMBTU, digits=3)
+
+    if !isempty(p.techs.steam_turbine) && p.s.ashp.can_supply_steam_turbine
+        @expression(m, ASHPToSteamTurbine[ts in p.time_steps], sum(m[:dvThermalToSteamTurbine]["ASHP",q,ts] for q in p.heating_loads))
+        @expression(m, ASHPToSteamTurbineByQuality[q in p.heating_loads, ts in p.time_steps], m[:dvThermalToSteamTurbine]["ASHP",q,ts])
+    else
+        ASHPToSteamTurbine = zeros(length(p.time_steps))
+        @expression(m, ASHPToSteamTurbineByQuality[q in p.heating_loads, ts in p.time_steps], 0.0)
+    end
+    r["thermal_to_steamturbine_series_mmbtu_per_hour"] = round.(value.(ASHPToSteamTurbine) / KWH_PER_MMBTU, digits=3)
```

Comment:
Wondering about needing the option or result of "ASHP to steamturbine" when that would never practically happen. Is there any value in have that always as zeros as opposed to not existing?